db/migrate/20241007155255_add_assignable_flag_to_school.rb

```diff
@@ -0,0 +1,5 @@
+class AddAssignableFlagToSchool < ActiveRecord::Migration[5.2]
+  def change
+    add_column :schools, :has_assignable_contacts, :boolean
```

Comment:
I feel like this table is small enough that you could probably set `null: false, default: false`

Reply:
I know I just said the above but now I'm thinking... is Salesforce guaranteed to return a boolean here? Just need to make sure it won't try setting this to nil.

Reply:
If it's cast as a boolean [here](https://github.com/openstax/openstax_salesforce/blob/main/lib/openstax/salesforce/remote/school.rb#L19), shouldn't this always return true/false?

Reply:
No idea 😂 have to check active_force...
Seems like it always returns a boolean: https://github.com/openstax/active_force/blob/3210375b8ea95b576c876de87f1e02fcdc0139d0/lib/active_force/mapping.rb#L64

Reply:
LOL, I looked at the same thing.. put the comment on the wrong thread 🤦🏼 
https://github.com/openstax/accounts/pull/1261#discussion_r1815361600